src/components/BreadCrumb/BreadCrumb.jsx

```diff
@@ -0,0 +1,2 @@
+
```

Comment:
can this file be deleted? i think it's also causing a merge conflict

Reply:
yes the breadcrumb shld've been deleted on an earlier PR (wasn't approved at that point) but in the later PR it wasn't (as git pull was still retrieving that file) hence the conflicts T-T